.github/CODEOWNERS

```diff
@@ -0,0 +1,16 @@
+# This is a comment.
+# Each line is a file pattern followed by one or more owners.
+# Order is important; the last matching pattern takes the most
+# precedence.
+
+# Everything in ufs-weather-model should be owned by the UFS Community Gatekeepers
+* @ufs-community/ufs-gatekeepers
```

Comment:
I see we have 7 ufs gate keepers covering different ufs model (not sure why), would they be each individual model onwer?,

Reply:
when a PR is issued, you will see ufs-gatekeepers as the reviewer. All 7 people will be notified that a review is requested. If one of them approves, that satisfies the review.
If you wish, we can remove ufs-gatekeepers from the weather model. But since, currently, ufs has multiple models e.g. weather-model and s2s-model and hafs-model, this would atleast alert a PR has been issued.

That is how this works.

Let me know and I can remove it, or we can do a trial and see what works and what doesn't.

Reply:
I think ufs-wm-gatekeepers is enough.

Reply:
done.